docs/docs/index.md

```diff
@@ -5,21 +5,20 @@ hide:
 
 # Firefly III documentation
 
-Welcome. This site contains all the documentation for **Firefly III** and the **Data Importer**. Please follow the links below or use the top navigation bar to go to the section of your choice. When in doubt, please use the search in the top right-hand corner.
+Welcome. This site contains all the documentation for **Firefly III** and the **Data Importer**. Please follow the links below or use the top navigation bar to go to the section of your choice. When in doubt, please use the search bar in the top right-hand corner.
 
 ## [Tutorials](tutorials/index.md)
+Do you want to learn the basics of how to use **Firefly III** and the **Data Importer**? Check out the [tutorials](tutorials/index.md) section to find such information as:
 
-Want to learn something new? Check out some excellent [tutorials](tutorials/index.md) for beginners and advanced users, such as:
-
-- [Set up your first set of bank accounts](tutorials/finances/first-steps.md)
-- [Introduce piggy banks to save money](tutorials/finances/piggy-bank.md)
-- [Predict expenses using bills](tutorials/finances/bill.md)
+- [How to set up your bank account(s)](tutorials/finances/first-steps.md)
```

Comment:
Let's not call tutorials "how to's".

Reply:
It does show you how to do that thing, though. While I can understand wanting to not use the "how to" language as to not confuse it with the how-to guides, I do think there should be more standardized language for these descriptions, which is what I was aiming for. I also wanted to make changes to most of these not just for the sake of standardization, but grammar, as if you, for example, broke down one of the bullet points into a sentence combined with the preamble before it, it makes no sense.

To illustrate my point, this:
Check out some excellent tutorials for beginners and advanced users, such as:

- Introduce piggy banks to save money](https://docs.firefly-iii.org/tutorials/finances/piggy-bank/


Turns into this:

Check out some excellent tutorials for beginners and advanced users, such as introduce piggy banks to save money.


That wording makes no sense if you read the preamble and subsequent list as a sentence, which is how I imagine most people naturally would. Now, this isn't the end of the world, and it doesn't make the documentation unreadable or anything, but it's still annoying and improper nonetheless. Maybe we could, instead of using the "How to X" language for each one, add suffixes and whatnot where necessary. For example, turning the "introduce" in "introduce piggy banks..." into "introducing" so it reads as "such as introducing piggy banks..."


(Sorry if this reply is a little difficult to understand, the concepts I was trying to convey are kind of complex in structure and origin and therefore were quit hard to put into the right words.)

Reply:
> as if you, for example, broke down one of the bullet points into a sentence combined with the preamble before it, it makes no sense.

That's an excellent point. I can merge this but I'll think of the best way to describe this.